src/System.Data.SqlClient/tests/ManualTests/SQL/SqlNotificationTest/SqlNotificationTest.cs

```diff
@@ -51,7 +51,7 @@ public void Test_DoubleStart_SameConnStr()
             Assert.True(SqlDependency.Stop(_startConnectionString), "Failed to stop listener.");
         }
 
-        [CheckConnStrSetupFact]
+        //[ConditionalFact(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringsSetup))]
```

Comment:
Why did you comment this out? 
This disables the test completely

Reply:
The same comment applies to the rest of the file.

Reply:
Oops. fixed.